src/en/articles/accessible-svg.md

```diff
@@ -1,143 +1,176 @@
 
-<pre> <code class="html">
+<pre><code class="html" title="html">
 &lt;button&gt;
-  &lt;svg focusable="false" aria-hidden="true"&gt; &lt;! --...--&gt; &lt;/svg&gt;
-  &lt;span class="visually-hidden"&gt; Search &lt;/ span&gt;
-&lt;/ button&gt;
-
-&lt;a href="/ Search"&gt;
-  &lt;svg focusable="false" aria-hidden="true"&gt; &lt;! --...--&gt; &lt;/svg&gt;
-  &lt;span class="visually-hidden"&gt; Search &lt;/span&gt;
-&lt;/ a&gt;
-</code> </pre>
+  &lt;svg aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
+  &lt;span class="visually-hidden"&gt;Search&lt;/span&gt;
+&lt;/button&gt;
 
-The visually-hidden class is used to hide the text except to technical help users (accessible masking).
+&lt;a href="/Search"&gt;
+  &lt;svg aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
+  &lt;span class="visually-hidden"&gt;Search&lt;/span&gt;
+&lt;/a&gt;
+</code></pre>
 
-or
+It is also possible to use the `aria-label` attribute:
 
-<pre> <code class="html">
+<pre><code class="html" title="html">
 &lt;button aria-label="Search"&gt;
-  &lt;svg focusable="false" aria-hidden="true"&gt; &lt;! --...--&gt; &lt;/svg&gt;
-&lt;/ button&gt;
-
-&lt;a href="/ Search" aria-label="Search"&gt;
-  &lt;svg focusable="false" aria-hidden="true"&gt; &lt;! --...--&gt; &lt;/svg&gt;
-&lt;/ a&gt;
-
-or, but much more risky ...
+  &lt;svg aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
+&lt;/button&gt;
 
-&lt;button&gt;
-  &lt;svg aria-labelledby="search-icon-title" focusable="false" role=”img”&gt;
-    &lt;title id="search-icon-title"&gt;Search&lt;/title&gt;
-    &lt;! --...--&gt;
-  &lt;/svg&gt;
-&lt;/ button&gt;
-</code> </pre>
+&lt;a href="/Search" aria-label="Search"&gt;
+  &lt;svg aria-hidden="true"&gt;&lt;!--...--&gt;&lt;/svg&gt;
+&lt;/a&gt;
+</code></pre>
 
 ## Last words ...
 
 In short, one tip for accessible SVG would be TEST and test again!
 
 ## Ressources (articles that have helped me a lot)
 
-- https://www.deque.com/blog/creating-accessible-svgs/
-- https://weboverhauls.github.io/demos/svg/
-- https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html 
-- https://www.slideshare.net/webaxe/svg-icons-and-screen-reader-accessibility 
-- https://github.com/JeremiePat/svg-accessible/blob/master/slides.md 
-- https://www.sarasoueidan.com/blog/accessible-icon-buttons/ 
\ No newline at end of file
+<a href="https://www.deque.com/blog/creating-accessible-svgs/">Creating Accessible SVGs - Deque</a>
+<a href="https://weboverhauls.github.io/demos/svg/">Accessible SVG test page</a>
+<a href="https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html">Contextually Marking up accessible images and SVGs - Scott O'Hara</a>
+<a href="https://www.slideshare.net/webaxe/svg-icons-and-screen-reader-accessibility">SVG Icons and Screen Reader Accessibility</a>
+<a href="https://github.com/JeremiePat/svg-accessible/blob/master/slides.md">SVG Accessible ! - Github Jeremie Patonnier</a>
```

Comment:
Change the link for the english version; https://github.com/JeremiePat/svg-accessible/blob/master/en/slides.md